Gem/Transfer/Scenes/AmountScene.swift

```diff
@@ -121,19 +123,21 @@ struct AmountScene: View {
                                 )
                             } secondary: {
                                 AnyView(
-                                    Button(Localized.Transfer.max, action: useMax)
-                                        .buttonStyle(ColorButton.lightGray(paddingHorizontal: 16, paddingVertical: 8))
-                                        .fixedSize()
+                                    HStack {
+                                        Button(Localized.Transfer.max, action: useMax)
+                                            .buttonStyle(ColorButton.lightGray(paddingHorizontal: 16, paddingVertical: 8))
```

Comment:
Instead of 16,8 use `Spacing.small`, `Spacing.medium`

Reply:
done